daprdocs/content/en/operations/resiliency/policies/retries/retries-overview.md

```diff
@@ -0,0 +1,148 @@
+| `duration` | Determines the time interval between retries. Only applies to the `constant` policy.<br/>Valid values are of the form `200ms`, `15s`, `2m`, etc.<br/> Defaults to `5s`.|
+| `maxInterval` | Determines the maximum interval between retries to which the [`exponential` back-off policy](#exponential-back-off-policy) can grow.<br/>Additional retries always occur after a duration of `maxInterval`. Defaults to `60s`. Valid values are of the form `5s`, `1m`, `1m30s`, etc |
+| `maxRetries` | The maximum number of retries to attempt. <br/>`-1` denotes an unlimited number of retries, while `0` means the request will not be retried (essentially behaving as if the retry policy were not set).<br/>Defaults to `-1`. |
+| `matching.httpStatusCodes` | Optional: a comma-separated string of [HTTP status codes or code ranges to retry](#retry-status-codes). Status codes not listed are not retried.<br/>Valid values: 100-599, [Reference](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)<br/>Format: `<code>` or range `<start>-<end>`<br/>Example: "429,501-503"<br/>Default: empty string `""` or field is not set. Retries on all HTTP errors. |
+| `matching.gRPCStatusCodes` | Optional: a comma-separated string of [gRPC status codes or code ranges to retry](#retry-status-codes). Status codes not listed are not retried.<br/>Valid values: 0-16, [Reference](https://grpc.io/docs/guides/status-codes/)<br/>Format: `<code>` or range `<start>-<end>`<br/>Example: "1,501-503"<br/>Default: empty string `""` or field is not set. Retries on all gRPC errors. |
+
+
+## Exponential back-off policy
+
+The exponential back-off window uses the following formula:
+
+```
+BackOffDuration = PreviousBackOffDuration * (Random value from 0.5 to 1.5) * 1.5
+if BackOffDuration > maxInterval {
+  BackoffDuration = maxInterval
+}
+```
+
+## Retry status codes
+
+When applications span multiple services, especially on dynamic environments like Kubernetes, services can disappear for all kinds of reasons and network calls can start hanging. Status codes provide a glimpse into our operations and where they may have failed in production. 
+
+### HTTP
+
+The following table includes some examples of HTTP status codes you may receive and whether you should or should not retry certain operations.
+
+| HTTP Status Code        | Safe to retry?         | Description                  |
+| ----------------------- | ---------------------- | ---------------------------- |
+| 404 Not Found           | ❌ No                  | The resource doesn't exist.  |
+| 400 Bad Request         | ❌ No                  | Your request is invalid.     |
+| 401 Unauthorized        | ❌ No                  | Try getting new credentials. |
+| 503 Service Unavailable | ✅ Yes                 | Service might recover.       |
+| 504 Gateway Timeout     | ✅ Yes                 | Temporary network issue.     |
+
```

Comment:
Can you add these as good http defaults too please? 

401...

408 Request Timeout | YES |  The server timed out waiting for the request.
429 Too Many Requests  | YES |  (Respect the Retry-After header if present).
500 Internal Server Error | YES | the server encountered an unexpected condition.
502 Bad Gateway  | YES | A gateway or proxy, received an invalid response.

503...